ts/state/ducks/settings.tsx

```diff
@@ -1,8 +1,7 @@
 import { isBoolean } from 'lodash';
 
 import { PayloadAction, createSlice } from '@reduxjs/toolkit';
-import { SettingsKey } from '../../data/settings-key';
-import { Storage } from '../../util/storage';
+import { SettingsKey } from '../../data/settings-key'; // ok: does not import anything else
```

Comment:
Are you just saying that this import is safe but don't import any other things from this file?

Reply:
removed